.travis.yml

```diff
@@ -22,5 +22,6 @@ install:
   - cd $OLDPWD
 mono:
   - 4.0.5
+osx_image: xcode7.1
 script:
-  - ./build.sh --quiet verify
+  - ./build.sh verify
```

Comment:
Travis output:

    Usage: ./build.sh -m [makefile] -n [nuget] [ [targets] ]
		   -m [makefile]  The makefile.shade to execute
		   -n [nuget]     nuget.exe
		   [targets]      A space separated list of targets to run
	The command "./build.sh verify" exited with 1.
	Done. Your build exited with 1.